lib/ddtrace/utils.rb

```diff
@@ -3,6 +3,11 @@
 module Datadog
   # Utils contains low-level utilities, typically to provide pseudo-random trace IDs.
   module Utils
+    # Empty immutable hash to be used.
+    # Useful to avoid allocations when the hash won't be modified:
+    # e.g. methods with empty +options={}+ as default.
+    EMPTY_HASH = {}.freeze
```

Comment:
Since this is effectively a constant, should we move this also to `Ext`?

I'm pretty sure we're going to apply what you've done here in many other places as well, and I think it might be better to have those avoid referencing `Utils` as its more of a function library vs `Ext` which is for universal constants.

Minor issue either way.